lib/buffer.js

```diff
@@ -21,7 +21,19 @@
 
 'use strict';
 
-const { Math, Object } = primordials;
+const {
+  Object: {
+    defineProperties,
+    defineProperty,
+    setPrototypeOf,
+    create
+  },
+  Math: {
+    floor,
+    trunc,
+    min
```

Comment:
Just a suggestion: At least for `trunc` or `create`, it might not be immediately obvious at the call site that it refers to `Math.trunc` or `Object.create`.

If you want, feel free to use more explicit named in those cases, e.g. `MathTrunc` or `ObjectCreate`.

Reply:
Great suggestion! I'll make the improvement tomorrow. Thank you again.

Reply:
Improvement done. So appreciated for your time.